### PR TITLE
Fix/cc 202

### PR DESCRIPTION
**Pull request by Steve-Rozeboom**
Updates based on feedback
Includes new footer CC-204

### CODE REVIEW

File: apps/common-capabilities/src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  GoAAppFooterMetaSection,
+  GoAAppFooterNavSection,
+} from '@abgov/react-components-4.20.2';
+
+const Footer = () => {
+  return (
+    <>
+      <GoAAppFooter maxContentWidth="1500px">
+        <GoAAppFooterNavSection maxColumnCount={1}>
+          <a href="/gettingstarted/index.html">Getting started</a>
```
Comment (smettu007) [inline apps/common-capabilities/src/components/Footer.tsx:13]: can you add `#getting-started` to the end of url? like this `/gettingstarted/index.html#getting-started`
so when the users land on this page, getting-started on the sidebar will be highlighted

**Comment by Steve-Rozeboom** [comment]
screencapture-localhost-3000-1gx-integration-legacy-imagis-index-html-2024-05-29-14_40_12

File: apps/common-capabilities/src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <>
+      <GoAAppFooter maxContentWidth="1500px">
+        <GoAAppFooterNavSection maxColumnCount={1}>
+          <a href="/gettingstarted/index.html#getting-started">Getting started</a>
+          <a href="/about/index.html">About</a>
+          <a href="/ecosystem/index.html">Eco-system</a>
+        </GoAAppFooterNavSection>
+
+        <GoAAppFooterNavSection>
+          <a href="/services/index.html">Services</a>
```
Comment (smettu007) [inline apps/common-capabilities/src/components/Footer.tsx:19]: can you move these two links to the above footer section?

Comment (Steve-Rozeboom) [reply apps/common-capabilities/src/components/Footer.tsx:19]: This is using Zion's first option. No need to move links

**Review by smettu007 — APPROVED** [review]

**Pull request merged**